### ISSUE TITLE
Clarification on Negative Pair Selection for NTXentLoss Function

### ISSUE THREAD

**Issue by ZhouMengjie** [lightly-ai/lightly#1187]
Dear Contributor,

Regarding the NTXentLoss function in your code, I am trying to understand the negative pair selection process. Given a group of queries [0,1,2,3,4] and a group of references [0_,1_,2_,3_,4_], could you clarify which samples are considered as negative examples for sample 0? Are [1_,2_,3_,4_] the only negative examples or is it also [1,2,3,4,1_,2_,3_,4_]? Thank you for your help in clarifying this.

**Comment by guarin** [comment]
Hi, thanks for using Lightly!

All negative samples are used ([1,2,3,4,1_,2_,3_,4_]). In the code we calculate `logits_00`, `logits_01`, `logits_10`, and `logits_11` here: https://github.com/lightly-ai/lightly/blob/00461d139d2a4280d91cbbc0d4d32d8766ef4227/lightly/loss/ntx_ent_loss.py#L154-L157

They correspond to the following:
* `logits_00`: [0,1,2,3,4] with [0,1,2,3,4] as negatives
* `logits_01`: [0,1,2,3,4] with [0_,1_,2_,3_,4_] as negatives
* `logits_10`: [0_,1_,2_,3_,4_] with [0,1,2,3,4] as negatives
* `logits_11`: [0_,1_,2_,3_,4_] with [0_,1_,2_,3_,4_] as negatives

Hope this helps :)
